yodeploy/test_integration/samples/nginx-app/deploy/hooks.py

```diff
@@ -0,0 +1,20 @@
+import getpass
+
+import mock
+
+from os.path import join
+
+from yodeploy.hooks.nginx import NginxHostedApp
+from yodeploy.test_integration.helpers import tests_dir
+
+
+class Hooks(NginxHostedApp):
+    # these overrides are only needed for testing
+    nginx = mock.Mock()
+    logs_path = join(tests_dir, 'filesys/var/log/nginx')
```

Comment:
I think you better use `os.path.join` instead.

Reply:
It is the same function.

Reply:
Oh, it is imported from `os`. I wouldn't to do this, it is easy to mix up with string `''.join()`